cpp/cmake/thirdparty/CUDF_GetRMM.cmake

```diff
@@ -32,6 +37,7 @@ function(find_and_configure_rmm VERSION)
                         "BUILD_BENCHMARKS OFF"
                         "CUDA_STATIC_RUNTIME ${CUDA_STATIC_RUNTIME}"
                         "CMAKE_CUDA_ARCHITECTURES ${CMAKE_CUDA_ARCHITECTURES}"
+                        "SUPPORTED_CUDA_ARCHITECTURES ${CMAKE_CUDA_ARCHITECTURES}"
```

Comment:
Why do we need to pass `SUPPORTED_CUDA_ARCHITECTURES` here?

Reply:
Oh we don't. Was testing something and forgot to remove it.